src/module.c

```diff
@@ -9463,16 +9464,6 @@ int moduleRegisterApi(const char *funcname, void *funcptr) {
 /* Global initialization at Redis startup. */
 void moduleRegisterCoreAPI(void);
 
-/* Some steps in module initialization need to be done last after server
- * initialization.
- * For example, selectDb() in createClient() requires that server.db has
- * been initialized, see #7323. */
-void moduleInitModulesSystemLast(void) {
```

Comment:
maybe it's a good idea to keep this function here (empty) for future use?
so that if it's needed later, someone doesn't introduce the call in the wrong place (like it did in the past)

Reply:
ok put it back